content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,194 @@
+   :alt: View of CRM leads and the option to schedule an activity.
+
+.. note::
+   Depending on the activity type, the :guilabel:`Schedule` button may be replaced by a
+   :guilabel:`Save` button, or an ::guilabel:`Open Calendar` button.
+
+Scheduled activities are added to the chatter for the record under :guilabel:`Planned activities`.
+
+.. image:: plan_activities/chatter-activities.png
+   :align: center
+   :alt: View of CRM leads and the option to schedule an activity.
+
+Activities can also be scheduled from the kanban, list, or activities view of an application.
+
+.. tabs::
 
-Plan activities
-===============
+   .. tab:: Kanban view
 
-Activities can be planned and managed from the chatter by clicking on *Schedule activity*,
-or through Kanban views.
+      Select a record on which to schedule an activity. Click on the :guilabel:`🕘 (clock)` icon,
+      then :guilabel:`Schedule An Activity`.
 
-.. image:: plan_activities/schedule_activity.png
+    .. image:: plan_activities/schedule-kanban-activity.png
+       :align: center
+       :alt: Kanban view of the CRM pipeline and the option to schedule an activity.
+
+   .. tab:: List view
+
+      Select a record on which to schedule an activity. Click on the :guilabel:`🕘 (clock)` icon,
+      then :guilabel:`Schedule An Activity`. If the record already has an activity scheduled, the
+      clock icon may be replaced by a :guilabel:`📞 (phone)` or an :guilabel:`✉️ (envelope)` icon.
+
+    .. image:: plan_activities/schedule-list-activity.png
+       :align: center
+       :alt: List view of the CRM pipeline and the option to schedule an activity.
+
+   .. tab:: Activity view
+
+      Select a record on which to schedule an activity. Move across the row to find the desired
+      activity type, then click the :guilabel:`＋ (plus sign)`.
+
+    .. image:: plan_activities/schedule-activity-view.png
+       :align: center
+       :alt: Activity view of the CRM pipeline and the option to schedule an activity.
+
+View scheduled activities
+=========================
+
+To view scheduled activities, click the :guilabel:`🕘 (clock)` icon on the menu bar. Doing so opens
+the :guilabel:`Activities menu`. Activities are separated out by the application where they were
+created, and by deadline. Select an icon to see the activities for that specific application.
+
+.. image:: plan_activities/activities-menu.png
    :align: center
-   :height: 300
-   :alt: View of crm leads and the option to schedule an activity for Odoo Discuss
+   :alt: View of CRM leads page emphasizing the activities menu.
 
-Set your activity types
------------------------
+Configure activity types
+========================
 
-A number of activities types are available by default in Odoo (call, email, meeting, etc.). However,
-you can set new ones going to :menuselection:`Settings --> Activity types`.
+To configure the types of activities in the database, go to :menuselection:`Settings application -->
+Activities --> Activity Types`.
```

Comment:
In the 15.0 runbot I'm using for this review, the 'Activities' setting (with Activity Types link) is located in the Discuss section. However, in the screenshot being used, it appears to be right up top in a 'general settings' area. I can see that this is because 'Activities' was written in the search bar. So, perhaps, it would be beneficial to include that instruction (typing 'Activities' in the search bar) to match the screenshot. Or, maybe it would be best to describe how it would appear without doing that. Just something to consider.

Reply:
This originally started as a CRM focused doc, however since it is listed under discuss/productivity, I tried to make it more general with the thought that I could then go back and create a more specified CRM focused one.